src/rabbit_autoheal.erl

```diff
@@ -381,6 +381,11 @@ restart_loser(State, Winner) ->
                       not_healing;
                   autoheal_safe_to_start ->
                       State
+                  after 60000 ->
+                      rabbit_log:warning(
+                          "Autoheal: Timed out waiting for autoheal_safe_to_start, 
+                           winner is ~p; Start and retry autoheal", [Winner]),
```

Comment:
I don't understand the "start and retry autoheal" part. It makes it sound like this is a recommendation to the operator.

Reply:
Start rabbit as normal, and "autoheal" will happen again because of "inconsistent_database".
This retry will probably succeed, because timeout is very unlikely to happen.